internal/broker/bindings/gardener_request_bindingmanager.go

```diff
@@ -0,0 +1,50 @@
+package broker
+
+import (
+	"context"
+	"fmt"
+
+	authenticationv1alpha1 "github.com/gardener/gardener/pkg/apis/authentication/v1alpha1"
+	shoot "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	"github.com/kyma-project/kyma-environment-broker/internal"
+	"github.com/kyma-project/kyma-environment-broker/internal/ptr"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+type GardenerBindingManager struct {
+	tokenExpiration int
```

Comment:
Wouldn't be it convenient to embed unit in the name? `tokenExpirationInSeconds`?

Reply:
Corrected.